README.md

```diff
@@ -14,6 +14,19 @@ This repository is a Drupal client module that can connect to a contentpool
 server. You'll need a drupal project for installing it. Refer to "Installation"
 for details.
 
+## Dependencies
+
+For the contentpool filter selection, there is a dependency on Vue 2.0, the
+version is setup via /admin/config/development/vuejs or in the config file, eg.: 
+
+```yaml
+libraries:
+  vue:
+    installation: cdnjs
```

Comment:
We need to make it work out-of-the-box as far as possible, to ease installation. Is there no reasonable default config by the module? If this is necessary, please add it to the module install config. Then it's enough to shortly mention it here imo.